src/button/template/content.json

```diff
@@ -3342,15 +3343,15 @@
         "es": {
             "checkout": "Pagar con {logo:pp} {logo:paypal}",
             "safer_tag": "La forma r\u00e1pida y segura de pagar",
-
             "later_tag": "Compre ahora y pague m\u00e1s adelante.",
             "pay": "Pagar con {logo:paypal}",
-            "buynow": "{logo:pp} {logo:paypal} Comprar ahora"
+            "buynow": "{logo:pp} {logo:paypal} Comprar ahora",
+            "installment": "Pagos sin<br>inter\u00e9s",
+            "installment_period": "Pague hasta<br>[installmentperiod]x sin inter\u00e9s"
```

Comment:
I wonder if instead of introducing a new style of placeholder here (`[foo]` vs `{foo}`) which might cause confusion, like "what's the difference between the two styles?", can we use `{installmentperiod}`? Then we can just define another function which we pass to `jsxRender` which handles the `installmentperiod` case.

Reply:
The square brackets here denote the `dynamic content` which is being plugged in. Now, if we replace [] with {}, we will need an additional specific function to be passed to the jsxRender. Whereas, in the present scenario, we are handling the replacement of foo [foo] inside context function itself. It picks up the foo from dynamicContent object without having to explicitly call it. What if we have tens of different dynamic content? Then it might not be a good idea to feed different functions for the respective contents to jsxRender.
As for the scalability, all we got to take care of is feeding the dynamicCOntent obj with different value and the [foo] would be replaced automatically in the content func.

Reply:
https://github.com/vishakha94/paypal-checkout/blob/latam/src/button/template/componentTemplate.jsx#L128-L133